site/faq.md

```diff
@@ -96,8 +96,13 @@ explanation.
 
 ### Why does Flux need a deploy key?
 
-Flux needs a deploy key to be allowed to push to the version control
-system in order to read and update the manifests.
+There are a number of Flux commands and API calls which will update the git repo in order to
+reflect the state of the cluster after applying the command. For more information about flux
+commands see [the fluxctl docs](./using.md).
+
+### Does Flux automatically sync changes back to git?
+
+No. It only syncs with git when a flux command or API call makes an applicable change.
```

Comment:
The second sentence seems to qualify the first, by saying ".. only when..". To make it  less ambiguous, perhaps elaborate by stating what _does_ happen -- that is, changes go from git to the cluster, and commands update git. Commits by people directly to the git repo will _also_ be applied to the cluster.